app/src/main/java/me/guillaumin/android/osmtracker/db/model/TrackStatistics.java

```diff
@@ -0,0 +1,195 @@
+package me.guillaumin.android.osmtracker.db.model;
+
+import me.guillaumin.android.osmtracker.db.TrackContentProvider;
+import me.guillaumin.android.osmtracker.db.TrackContentProvider.Schema;
+import android.content.ContentResolver;
+import android.database.Cursor;
+import android.os.Bundle; 
+
+/**
+ * Represents statistics, such as  total length and maximum speed, for a Track
+ *
+ * @author Arseniy Lartsev
+ *
+ */
+public class TrackStatistics {
+	private long trackId;
+	private long pointCount;
+	private float length;
+	private float maxSpeed;
+	private double lastLatitude;
+	private double lastLongitude;
+	private long lastTime;
+	private int lastId;
+	private float timeMoving;
+	private ContentResolver contentResolver;
+	
+	private static double square(double x) {
+		return x*x;
+	}
+
+	/**
+	 * Compute the approximate distance in meters between two locations, with 0.5% precision
+	 */
+	public static float getDistance(double lat1, double long1, double lat2, double long2) {
+		final double R = 6370e3; // Earth radius
+		final double rad_per_deg = Math.PI/180; // radians per degree
+		
+		lat1 *= rad_per_deg;
```

Comment:
It would be more Java-idiomatic to not modify the input variables (they're actually often declared `final`)